src/skimage/measure/_moments.py

```diff
@@ -255,7 +255,7 @@ def moments_central(image, center=None, order=3, *, spacing=None, **kwargs):
     float_dtype = _supported_float_type(image.dtype)
     if spacing is None:
         spacing = np.ones(image.ndim, dtype=float_dtype)
-    calc = image.astype(float_dtype, copy=False)
+    calc = image.astype(float_dtype)
```

Comment:
Do you think we have a problem with non-contiguity?

Reply:
Yup - almost certainly - see https://github.com/scikit-image/scikit-image/issues/7941#issuecomment-3536331449

Reply:
Oops, sorry, I meant to remove the copy change.   Have done now.